protodata/src/main/proto/spine/validation/rule.proto

```diff
@@ -19,18 +19,18 @@ import "google/protobuf/any.proto";
 // The second operand is the `other_value`. `other_value` is typically derived from the Protobuf
 // definition of the validation rule.
 //
-message Rule {
+message SimpleRule {
 
     // The field which is being compared.
     protodata.Field field = 1;
 
     oneof operation {
 
         // A simple comparison sign.
-        Sign sign = 2;
+        ComparisonOperator sign = 2;
 
         // A custom comparison method, which cannot be covered by a combination of `Sign`s.
-        UnknownOperation unknown_operation = 3;
+        CustomOperator unknown_operation = 3;
```

Comment:
I believe, with the `UnknownOperation` renamed to `CustomOperator` this field is now `custom_operator`?